.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,52 +1,67 @@
-## Pull Request (PR) Checklist
+## Pull request (PR) checklist
+
 Please check if your pull request fulfills the following requirements:
+
+<!-- 💡 Tip: Tick checkboxes like this: [✅]. -->
+
 - [ ] The PR is submitted to the `main` branch.
 - [ ] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) and my PR doesn't break the rules.
 - [ ] I've read and understand the [Developer Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Guidelines.md).
 - [ ] I confirm that I've run the code locally and everything works as expected.
-- [ ] 🎬 I've attached a **screen recoding** of the changes.
 - [ ] My PR includes only the necessary changes to fix the issue (i.e., no unnecessary files or lines of code are changed).
+- [ ] 🎬 I've attached a **screen recording** of using the new code to the next paragraph.
+
+
+## Your screen recording of the changes ❗IN USE❗:
+
+<!-- 💡 Tip: Drag & drop the video here. -->
+
 
-> Tip: drag & drop the video to the PR description.
 
 ## What's changed?
-<!--
-Tip: you can attach screenshots using a markdown table.
+Describe with a few bullets **what's new:**
 
-Before | After
---|--
-image1 | image2
+<!-- 💡 Tip: After each more important point leave one line empty and show your changes in markdown table with screenshots or screen recordings replacing {media}. In the end, it should look like this:
+
+- I've fixed...
+
+Before|After
+-------|-------
+{media}|{media}
+{media}|{media}
 -->
 
-Describe with a few bullets **what's new:**
-- a
-- b
-- c
 
-> 💡 Tip: Please, attach screenshots and screen recordings. It helps a lot!
+- ...
+
 
-## Risk Factors
+- ...
+
+
+## Risk factors
 
 **What may go wrong if we merge your PR?**
 
-- a
-- b
+- ...
+- ...
 
-**In what cases your code won't work?**
+**In what cases won't your code work?**
 
-- a
-- b
+- ...
+- ...
 
-## Does this PR closes any GitHub Issues?
 
-Check **[Ivy Wallet Issues](https://github.com/Ivy-Apps/ivy-wallet/issues)**.
+## Does this PR close any GitHub issues?
 
-- Closes #ISSUE_NUMBER
+- Closes #{ISSUE_NUMBER}
```

Comment:
People always get confused about this one. I'm concerned about them writing 

Closes #{123}

Reply:
In the comment there's already written that if you do it correctly you will see the issue title pop up, so I'm just going to leave an example of how it should look like but keep the style - if it's not going to work then it will be patched in the future I think.